apps/web/src/SettingsRoutes.tsx

```diff
@@ -0,0 +1,92 @@
+import { Cloud, IconLockPerson, SSO, UserAccess } from '@novu/design-system';
+import { FeatureFlagsKeysEnum } from '@novu/shared';
+import { Route } from 'react-router-dom';
+import { ProductLead } from './components/utils/ProductLead';
+import { ROUTES } from './constants/routes.enum';
+import { useFeatureFlag } from './hooks';
+import { BillingRoutes } from './pages/BillingPages';
+import { BrandingForm } from './pages/brand/tabs';
+import { MembersInvitePage as MembersInvitePageNew } from './pages/invites/v2/MembersInvitePage';
+import { SettingsPage } from './pages/settings/SettingsPage';
+import { ApiKeysCard } from './pages/settings/tabs';
+import { EmailSettings } from './pages/settings/tabs/EmailSettings';
+
+/** Note: using a hook is the only way to separate routes */
```

Comment:
Other way will be use `Routes` to wrap the `Route` and return it as `<SettingsRoutes/>`.
We do something similar in our enterprise packages.

Reply:
also another way will be to  migrate to [useRoutes](https://reactrouter.com/en/main/hooks/use-routes) hook of `react-router-dom`. It will enable us to use objects to define the routes.